agent-control/src/agent_control/defaults.rs

```diff
@@ -6,12 +6,16 @@ use opamp_client::opamp::proto::{AgentCapabilities, CustomCapabilities};
 use opamp_client::operation::capabilities::Capabilities;
 
 pub const AGENT_CONTROL_ID: &str = "agent-control";
+pub const AGENT_CONTROL_CD_ID: &str = "agent-control-cd";
```

Comment:
It is not needed anymore, is it? I'm not sure if there is already a constant for the default value for such HelmRelease name 🤔 (It is not an agent-id anyway)

Reply:
I think we should reserve the name anyway, and indeed there's a constant defined elsewhere, I'll fine-tune it!

Reply:
I can move them to the CLI and aggregate them as reserved in the slice anyways, doing that!

Reply:
Rewrote latest commit with more constant changes: https://github.com/newrelic/newrelic-agent-control/pull/1563/commits/1ebf0e2a326532170a81a7aaaacce97e5562c024